frontend/src/i18n/locales/en.yaml

```diff
@@ -424,7 +425,7 @@ manage:
     details:
       title: Video details
       share-direct-link: Share via direct link
-      set-time: 'Start at: '
+      set-time: 'Start video at (h/m/s): '
```

Comment:
I don't like this change. For users without visual limitations, the h/m/s information is very obvious from the input box already. I haven't tested it with a screen reader, but if the input is indeed never fully read or sth and it thus needs improvement, we need to do it differently that doesn't make it worse for other users.

Reply:
Agree. I can probably add some aria labels for screenreader purposes instead.